xmake/modules/package/manager/pacman/find_package.lua

```diff
@@ -238,6 +236,11 @@ function main(name, opt)
         if result.links then
             result.links = table.reverse_unique(result.links)
         end
+
+        -- We should get version from pacman, because if a pacman package contains multiples .pc we may get a wrong version
+        local version = try { function() return os.iorunv(pacman.program, {"-Q", name}) end }
```

Comment:
check version is nil. and use semver.match

https://github.com/xmake-io/xmake/blob/25c3526637322cdd36a0bc78b3cd7716dcc62382/xmake/modules/package/manager/nimble/find_package.lua#L40